src/app/[locale]/components/Header.tsx

```diff
@@ -1,15 +1,39 @@
 import { Navigation } from '@/app/[locale]/components/Navigation'
+import { Link, Logo } from '@/components/index'
 import Image from 'next/image'
 
-const Header = () => {
+export interface HeaderProps {
```

Comment:
Vanwaar de keus voor hier een nieuw element toewijzen aan logoElement in plaats van bijvoorbeeld:

```
const Header = ({ homepage, logo }: HeaderProps) => {
  const logoElement = (
    <Logo>
      <Image src={logo.src} alt={logo.label} width={275} height={150} />
    </Logo>
  );

  return (
    <header>
      {homepage ? (
        <Link boxContent href={homepage.href} aria-labelledby="logo-link-label">
          <span id="logo-link-label" hidden>
            {homepage.label}
          </span>
          {logoElement}
        </Link>
      ) : (
        logoElement
      )}
    </header>
  );
};
```

Is dit sneller oid?